packages/main/package.json

```diff
@@ -7,7 +7,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -b",
```

Comment:
I added a `-b` flag which causes any referenced projects to automatically rebuild. This way the common package will automatically build when we run `yarn start` or `yarn build` in the main package.